docs/packaging.md

```diff
@@ -0,0 +1,120 @@
+# Packaging
+
+There are multiple ways to use and package Robo scripts; a few of the alternatives are presented below.
+
+## Adding a RoboFile to your Project
+
+In its simplest form, Robo is used by creating a RoboFile.php at the root of your project.
+
+If `robo` is in your $PATH:
+```
+$ cd myproject
+$ robo mycommand
+```
+Alternately, add `robo` to your composer.json file:
+```
+$ cd myproject
+$ composer require consolidation/Robo:~1
+$ ./vendor/bin/robo mycommand
+```
+There are more examples of this in the [Getting Started](getting-started.md) guide.
+
+## Implementing Composer Scripts with Robo
+
+When using Robo in your project, it is convenient to define Composer scripts that call your Robo commands.  Simply add the following to your composer.json file:
+```
+{
+    "name": "myorg/myproject",
+    "require": {
+        "consolidation/Robo": "~1"
+    },
+    "scripts": {
+        "test": "composer robo test",
+        "phar": "composer robo phar:build",
+        "robo": "robo --ansi --load-from $(pwd)/scripts/BuildCommands.php"
+    }
+}
+```
+*Note*: When you include Robo as a library like this, some external projects used by certain core Robo tasks are not automatically included in your project.  See the `"suggest":` section of Robo's composer.json for a list of external projects you might also want to require in your project.
+
+Once you have set up your composer.json file (and ran `composer update` if you manually changed the `require` or `require-dev` sections), Composer will ensure that your project-local copy of Robo in the `vendor/bin` dir is in your $PATH when you run the additional Composer scripts that you declared:
+```
+$ cd myproject
+$ composer test
+$ composer phar
+```
+This will call the public methods `test()` and `phar()` in your RoboFile.php when using `composer test` and `composer phar`, respectively.
+
+Advertising your build commands as Composer scripts is a useful way to provide the key commands used for testing, building or packaging your application. Also, if your application should happen to provide a commandline tool to perform the operations of the application itself, then defining your build commands in their own RoboFile provides desirable separation, keeping your build commands out of the help and list commands of your primary script.
+
+If you would like to simplify the output of your script (e.g. when running on a CI service), replace the `--ansi` option in the example above with `--no-ansi`, and  colored terminal output and progress bars will be disabled. 
+
+## Creating a Standalone Phar with Robo
```

Comment:
And this can go to "Framework page". The idea is a step y step instructions on how to create custom CLI application using Robo as a framework. Don't hesitate to provide examples using your experience with Drush